daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md

```diff
@@ -98,7 +98,7 @@ The code inside the `process_batch` function is executed every 10 seconds (defin
 def process_batch():
 ```
 
-The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgres.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. 
+The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgresql.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. 
```

Comment:
we'll need to update this in [the quickstarts repo](https://github.com/dapr/quickstarts/tree/master/bindings/components)

Reply:
It's not a bad idea but it's not strictly required either. "postgres" continues to be available as alias